### PR TITLE
The Garden - A small botany ship

### PR DESCRIPTION
**Pull request by DustScoundrel**
## About the PR

I added the Garden, a small-sized botany ship with sequestered grow bays and a full horticulture setup. Designed for the green thumbs amongst us.

## Media

image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added the Garden, a small botany vessel.

### CODE REVIEW

**Comment by dvir001** [comment]
No PDA
It has free ID with permission in it

**Comment by DustScoundrel** [comment]
Ah, damn. I just wanted it there for flavor. I'll remove it

**Comment by DustScoundrel** [comment]
PDA has been removed. Now I just need to figure out how to make the change

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
PDA need to be indeed removed.

And remove the HS infront of the name as we removed tags from all ships to avoid looong names.

**Review by FoxxoTrystan — APPROVED** [review]

File: Resources/Prototypes/_NF/Shipyard/garden.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: Garden
+  name: HS Garden
+  description: A small botany vessel dedicated to horiticultural experimentation.
+  price: 29750
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/garden.yml:5]: it was only worth around 16000 when I spawned it in. Im gunna change this to be a tad cheaper, maybe make it just barely purcahaseable for a new character.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**